recipes/ulfius/all/test_package/test_package.c

```diff
@@ -0,0 +1,56 @@
+/**
+ * test.c
+ * Small Hello World! example
+ * to compile with gcc, run the following command
+ * gcc -o test test.c -lulfius
+ */
+#include <ulfius.h>
+
+#include <stdio.h>
+#include <string.h>
+
+#define PORT 8080
+
+/**
+ * Callback function for the web application on /helloworld url call
+ */
+int callback_hello_world (const struct _u_request * request, struct _u_response * response, void * user_data) {
+    ulfius_set_string_body_response(response, 200, "Hello World!");
+    return U_CALLBACK_CONTINUE;
+}
+
+/**
+ * main function
+ */
+int main(int argc, char *argv[]) {
+    struct _u_instance instance;
+
+    // Initialize instance with the port number
+    if (ulfius_init_instance(&instance, PORT, NULL, NULL) != U_OK) {
+        fprintf(stderr, "Error ulfius_init_instance, abort\n");
+        return(1);
+    }
+
+    // Avoid running a server on CI
+    if (argc > 1 && strcmp(argv[1], "run") == 0) {
+        // Endpoint list declaration
+        ulfius_add_endpoint_by_val(&instance, "GET", "/helloworld", NULL, 0, &callback_hello_world, NULL);
```

Comment:
I would prefer avoiding networking usage on CI if possible. You could use `ulfius_global_init` & `ulfius_global_close` instead.

Reply:
It's not run when no `run` argument is given.

Reply:
Oh wait, you probably meant the `ulfius_init_instance`.

Reply:
Fixed in [65f964f](https://github.com/conan-io/conan-center-index/pull/13650/commits/65f964f1e7d18c60562d334fe8bc35ba044fdcd7)
I kept the framework configuration because it is a good quick self-test.